arch/arm/soc/nordic_nrf/nrf52/soc.c

```diff
@@ -437,9 +57,6 @@ static int nordicsemi_nrf52_init(struct device *arg)
 
 	_ClearFaults();
 
-	/* Setup master clock */
-	clock_init();
```

Comment:
Same, will this work without this call?

Reply:
`SystemInit()` calls at the end `SystemCoreClockUpdate()` which does exactly this.

Reply:
Got it, thanks!